packages/notification/components/notifications/index.tsx

```diff
@@ -39,7 +45,7 @@ export const Notifications = ({
                 scale: reducedMotion ? 1 : 0.75,
                 transition: { duration: 0.2 },
               }}
-              role="status"
+              role={role}
```

Comment:
(Question 🙋🏻) Is it possible to auto-detect the role based on whether or not the notification contains interactive content?

Reply:
The thinking here was following the same reason we have explicit [Button](https://github.com/hashicorp/dev-portal/tree/main/src/components/button) and [ButtonLink](https://github.com/hashicorp/dev-portal/tree/main/src/components/button-link) components vs a single Button component that changes based on the attributes provided.